### ISSUE TITLE
Nest Integration - support multiple HA instances (e.g. prod, dev)

### ISSUE THREAD

**Issue by schuffr** [home-assistant/home-assistant.github.io#32109]
### Feedback

great write up. I'd love to a short section on how to manage integrations with dev and test HA instances. Following the same instructions on a second HA instance did not work. a short statement about whether this can work and if so, how to do it would be great!

### URL

https://www.home-assistant.io/integrations/nest/

### Version

2024.3.3

**Comment by allenporter** [comment]
Yes, this works, I do this and run a single nest home on multiple instances. I think the only differnece is you need to change the oauth url for my home assistant during the setup.

Can you elaborate on what you mean by "did not work."?
